package.json

```diff
@@ -125,170 +125,154 @@
   "runtimeDependencies": [
     {
       "id": "OmniSharp",
-      "description": "OmniSharp for Windows (.NET 4.6 / x86)",
-      "url": "https://download.visualstudio.microsoft.com/download/pr/47df6cf8-7648-4fff-9e9c-304f0202d31c/66f82f6333721c464316d7daa07bc115/omnisharp-win-x86-1.37.17.zip",
-      "fallbackUrl": "https://roslynomnisharp.blob.core.windows.net/releases/1.37.17/omnisharp-win-x86-1.37.17.zip",
-      "installPath": ".omnisharp/1.37.17",
+      "description": "OmniSharp for Windows (.NET 4.7.2 / x86)",
+      "url": "https://roslynomnisharp.blob.core.windows.net/releases/1.38.0/omnisharp-win-x86-1.38.0.zip",
+      "installPath": ".omnisharp/1.38.0",
       "platforms": [
         "win32"
       ],
       "architectures": [
         "x86"
       ],
-      "installTestPath": "./.omnisharp/1.37.17/OmniSharp.exe",
+      "installTestPath": "./.omnisharp/1.38.0/OmniSharp.exe",
       "platformId": "win-x86",
       "isFramework": true,
-      "integrity": "95ECBABF821ED45E4E6F3173E2165C5F593FC4611040475DBD6D54C16D78F4F8"
+      "integrity": "1E8B8EC8A69A5897017732442B90849393CA3FD2E5C33A406E25D8B065E5637E"
     },
     {
       "id": "OmniSharp",
       "description": "OmniSharp for Windows (.NET 6 / x86)",
-      "url": null,
-      "fallbackUrl": null,
-      "installPath": null,
+      "url": "https://roslynomnisharp.blob.core.windows.net/releases/1.38.0/omnisharp-win-x86-net6.0-1.38.0.zip",
+      "installPath": ".omnisharp/1.38.0-net6.0",
```

Comment:
This install path matches the path we use when downloading latest package and avoids conflicting with full framework installs.